source/extensions/access_loggers/open_telemetry/grpc_access_log_impl.cc

```diff
@@ -44,6 +35,32 @@ opentelemetry::proto::common::v1::KeyValue getStringKeyValue(const std::string&
 
 } // namespace
 
+GrpcAccessLoggerImpl::GrpcAccessLoggerImpl(
+    const Grpc::RawAsyncClientSharedPtr& client,
+    const envoy::extensions::access_loggers::open_telemetry::v3::OpenTelemetryAccessLogConfig&
+        config,
+    Event::Dispatcher& dispatcher, const LocalInfo::LocalInfo& local_info, Stats::Scope& scope)
+    : GrpcAccessLogger(
+          config.common_config(), dispatcher, scope, std::nullopt,
+          std::make_unique<Common::UnaryGrpcAccessLogClient<ExportLogsServiceRequest,
+                                                            ExportLogsServiceResponse>>(
+              client,
+              *Protobuf::DescriptorPool::generated_pool()->FindMethodByName(
+                  "opentelemetry.proto.collector.logs.v1.LogsService.Export"),
+              GrpcCommon::optionalRetryPolicy(config.common_config()),
+              [this, &dispatcher]() {
+                OTelLogRequestCallbacks* ptr = new OTelLogRequestCallbacks(
```

Comment:
This is mostly a nit. 
The lambda body is rather large and it makes the the constructor of UnaryGrpcAccessLogClient hard to read. It would be more readable to move this into a member function.